### DISCUSSION TITLE
Zenodo + GitHub linkage

### DISCUSSION THREAD

**Discussion by StanczakDominik** [PlasmaPy/PlasmaPy#1329] [General]
@rocco8773 I'm trying out that Zenodo + Github link you suggested. According to [this](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content), Zenodo will automatically archive a release (hell if I know based on what metadata!) when we do a release through https://github.com/PlasmaPy/PlasmaPy/releases/new, which we haven't been using.

So basically, looking at https://docs.plasmapy.org/en/stable/development/release_guide.html, I think I can skip the Zenodo/DOI stuff, go through the usual motions, make a tag and use that to mint a "GitHub release", which should automatically get up on Zenodo. I expect we should be able to amend any and all metadata afterwards; the only difference would be the lack of the reserved DOI in the actual files for the 0.7 release, which is probably not worth the effort to have them in there.

All in all, I think we can try it! @rocco8773  @namurphy does this look okay to you guys?

**Comment by namurphy** [comment]
This sounds good. I'm happy to try it out!

**Comment by StanczakDominik** [comment]
... I'll just copy-paste the email I've just written to the folks over at Zenodo:

> Hi! I was trying to release a new version of PlasmaPy, which we've previously manually archived here: https://zenodo.org/record/4602818. However, I tried using the new GitHub release integration and ended up with https://zenodo.org/record/5710242, which is completely disconnected from the previous record. That's not really what I was going for. However, I cannot modify the new record at all, neither to link it to the previous release nor to delete the new one and try again. I've since found out, looking through the docs here, https://developers.zenodo.org/#add-metadata-to-your-github-repository-release that I should have filled out the .zenodo.json file.
> Could I ask you to delete https://zenodo.org/record/5710242 and I'll manually add the new version for now, retrying the zenodo integration with a filled out metadata file for the next version?
> Thanks!

**Reply by StanczakDominik** [reply]
Now fixed at https://zenodo.org/record/5247589 . We'll still need to tinker with te .zenodo.json file a little so this can actually be done automatically the next time.
